topi/include/topi/transform.h

```diff
@@ -1309,5 +1309,57 @@ inline Tensor one_hot(const Tensor& indices, const PrimExpr on_value, const Prim
       name, tag);
 }
 
+/*!
+ * \brief Get a dense tensor.
+ * \param sparse_indices sparse_indices[i] contains the complete index where sparse_values[i] will be placed.
+ * \param sparse_values is a 0-D or 1-D tensor. Values corresponding to each row of sparse_indices
+ * \param default_value is a 0-D tensor. Defaults to zero.
+ * \param output_shape is the shape of the dense output tensor
+ * \param name output tensor name.
+ * \param tag output tensor tag.
+ * \return Tensor of output_shape.
+ */
+inline Tensor sparse_to_dense(const Tensor& sparse_indices,
+                              const Tensor& sparse_values,
+                              const Tensor& default_value,
+                              const Array<Integer>& output_shape,
+                              const std::string name = "T_sparse_to_dense",
+                              const std::string tag = kInjective) {
+  CHECK(sparse_indices->dtype.is_int())
+    << "sparse_indices only accepts integer values";
+
+  CHECK_LE(sparse_indices->shape.size(), 3)
+    << "sparse_indices tensor should be 0D, 1D, or 2D only";
+
+  CHECK_LE(sparse_values->shape.size(), 2)
+    << "sparse_values tensor should be 0D or 1D only";
+
+  const auto rank_sparse_indices = static_cast<int>(sparse_indices->shape.size());
```

Comment:
validate_indices flag has any change in behavior? if so, consider that as well. 
[Reference](https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/sparse/to_dense)

Reply:
@siju-samuel : validate_indices is a runtime check on data passed in. (no duplicate indexes, indexes should be sorted) So no change in behavior. Hence this is not considered.